themes/openy_themes/openy_carnation/templates/node/event/node--event--teaser.html.twig

```diff
@@ -104,9 +104,11 @@ set classes = [
       {{ title_suffix }}
     </div>
 
-    <div class="event-location">
-      <i class="fas fa-map-marker-alt mr-2 text-primary"></i> {{ content.field_event_location }}
-    </div>
+    {% if content.field_event_location|render|trim is not empty %}
```

Comment:
Seems like we need to use isEmpty() for Twig stability.
Please, verify https://github.com/ymcatwincities/openy/pull/1900/files

Reply:
it seems that isEmpty() don't work (on screen debug of provided PR)
http://i.imgur.com/RGkvHAy.png

Reply:
@manachynskyi you are trying to use isEmpty() method to the wrong object.
isEmpty() is a twig representation of the same entity method.
content is not an entity. This is render array, prepared for template.
This twig template has node object. You could use it.

Reply:
Thank you @anpolimus for the great collaboration